staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -319,6 +319,10 @@ func (o LogsOptions) RunLogs() error {
 		return err
 	}
 
+	if len(requests) == 0 {
+		return fmt.Errorf("no resources found in %s namespace", o.Namespace)
+	}
+
 	if o.Follow && len(requests) > 1 {
```

Comment:
@soltysh

Look at this, it counts pods in the same way I do. 
Do you think I change from this code to len(o.Object.(*corev1.PodList).Items) too?

Original code

https://github.com/kubernetes/kubernetes/blob/82baa26905c94398a0d19e1b1ecf54eb8acb6029/staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go#L316-L334

Changed code(Example)
```go
func (o LogsOptions) RunLogs() error {
    podCount := len(o.Object.(*corev1.PodList).Items)
    if o.Follow && podCount > 1 && podCount > o.MaxFollowConcurrency {
        return fmt.Errorf("you are attempting to follow %d log streams, but maximum allowed concurrency is %d, use --max-log-requests to increase the limit", len(requests), o.MaxFollowConcurrency,)
    }

    requests, err := o.LogsForObject(o.RESTClientGetter, o.Object, o.Options, o.GetPodTimeout, o.AllContainers)
    if err != nil {
        return err
    }

    if o.Follow && podCount > 1 {
        return o.parallelConsumeRequest(requests)
    }
    return o.sequentialConsumeRequest(requests)
}
```